### DISCUSSION TITLE
My github student discount is not yet activated

### DISCUSSION THREAD

**Discussion by monika3421** [community/community#52692] [GitHub Education]
I have uploaded id and also logged in from official email. Still it getting rejected automatically . How to fix that?

**Comment by samus-aran** [comment]
Hi @monika3421  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
